network/transparent_vlan_endpointclient_linux.go

```diff
@@ -513,15 +513,17 @@ func (client *TransparentVlanEndpointClient) DeleteEndpointsImpl(ep *endpoint, g
 
 	log.Printf("[transparent vlan] There are %d routes remaining after deletion", routesLeft)
 
-	if routesLeft <= numDefaultRoutes {
-		// Deletes default arp, default routes, vlan veth; there are two default routes
-		// so when we have <= numDefaultRoutes routes left, no containers use this namespace
```

Comment:
We are skipping deletion of vnet namespace if there are no containers but when there are new containers for that namespace, we won't need to create that namespace again ? Is that correct ?

Wanted to know if there is a scenario in which the namespaces are getting accumulated since they were not deleted.

Reply:
good question. cni creates netns per vlanid and at max vlanid can range from 1-4096 and also we dont support more than 150 containers in multitenancy. so not deleting netns should not cause any issue

vmss000000:/# cat /proc/sys/user/max_net_namespaces
27653